### ISSUE TITLE
'else' in multi-line 'guard' statement loses indentation

### ISSUE THREAD

**Issue by meherkasam** [nicklockwood/SwiftFormat#690]
@nicklockwood I found some weird behaviour around multi-line `guard` statements. Here's my test example:
```
class MyClass {
    struct MyStruct {
        private func doSomething() {
            guard let foo = blah,
                bar.isSomething else {
                doSomething()
            }
        }
    }
}
```

Running the following command:
` $ Pods/SwiftFormat/CommandLineTool/swiftformat test.swift --rules elseOnSameLine,wrapMultilineStatementBraces`

Expected outcome:
```
class MyClass {
    struct MyStruct {
        private func doSomething() {
            guard let foo = blah,
                bar.isSomething
            else {
                doSomething()
            }
        }
    }
}
```

Actual outcome:
```
class MyClass {
    struct MyStruct {
        private func doSomething() {
            guard let foo = blah,
                bar.isSomething
else {
                doSomething()
            }
        }
    }
}
```

**Comment by nicklockwood** [comment]
@meherkasam thanks for reporting, I'll get it fixed asap.

**Comment by nicklockwood** [comment]
I think this was probably masked in testing by the fact the `indent` rule normally fixes it automatically.

**Comment by meherkasam** [comment]
Yup, you're right!

Running the following command works fine:
`$ Pods/SwiftFormat/CommandLineTool/swiftformat test.swift --rules elseOnSameLine,wrapMultilineStatementBraces,indent`

I didn't have the `indent` rule turned on yet.

**Comment by nicklockwood** [comment]
@meherkasam fixed in 0.45.4.
